content/docs/reference/spec/migration/buildpack-api-0.5-0.6.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.5 -> 0.6"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.6) for buildpack API 0.6 for the full list of changes and further details.
+
+### Opt-in layer caching
```

Comment:
Should this and the fact that the layer types have been moved under a types table be two separate entries?

Reply:
I agree. It's also 2 separate entries in the spec: https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.6.

Reply:
I don't know... the fact that it's two separate issues is an artifact of how the RFC unfolded (see comment: https://github.com/buildpacks/spec/pull/151#issuecomment-708631142).

Reply:
However I do think the paragraph below could be re-worded...taking a look.

Reply:
How does it look now?